plugins_src/autouv/wpc_autouv.erl

```diff
@@ -19,7 +19,7 @@
 -include_lib("wings/src/wings.hrl").
 -include_lib("wings/e3d/e3d_image.hrl").
 -include("auv.hrl").
- 
```

Comment:
In the future if you do whitespace fixes on otherwise not edited lines, please do them in a separate commit.
Because now it is very hard to see what you have changed.

Reply:
I noticed that by checking _git diff_. That was made by VSCode - I didn't touch anything.

Reply:
I guess if you run the formatter it does it automatically.